torch/_inductor/mkldnn.py

```diff
@@ -200,20 +200,8 @@ def _update_module_params(self, conv_transpose, input_size):
 
     def _conv_transpose_forward(self, input, weight, bias):
         if self.padding_mode != "zeros":
-            return torch.ops.mkldnn._convolution_transpose_pointwise(
-                F.pad(
-                    input, self._reversed_padding_repeated_twice, mode=self.padding_mode
-                ),
-                weight,
-                bias,
-                _pair(0),
-                self.output_padding,
-                self.stride,
-                self.dilation,
-                self.groups,
-                "none",
-                [],
-                "",
+            raise ValueError(
+                "Only `zeros` padding mode is supported for PackedConvTranspose2d"
```

Comment:
Is this related to the issue https://github.com/pytorch/pytorch/issues/100344 you are trying to fix? Is it covered by UT?

Reply:
is it not related to the issue, the origin code is wrong, ConvTranspose only supports zero padding mode, see https://github.com/pytorch/pytorch/blob/main/torch/nn/modules/conv.py#L945.